### ISSUE TITLE
TypeError: '<' not supported between instances of 'tuple' and 'int'

### ISSUE THREAD

**Issue by Patrick5** [pdfminer/pdfminer.six#90]
I'm occasionally getting an error:

File "c:\Anaconda3\lib\site-packages\pdfminer.six-20170720-py3.6.egg\pdfminer\psparser.py", line 233, in fillbuf
    if self.charpos < len(self.buf):
TypeError: '<' not supported between instances of 'tuple' and 'int'

As far as I can tell the only place this might be caused by is line 350 in psparser.py
	            `return (self._parse_comment, len(s))`
where a tuple is in fact returned from function _parse_comment(self, s, i)

I hope this is enough info.
Patrick

**Comment by vstoykov** [comment]
Related (Duplicate?) of #89

**Comment by timb07** [comment]
Patrick's suggested fix is the same as the one in #109 (although that issue identifies the bug without describing how it manifests).

I've seen this problem also, and the fix seems to work for the PDFs I've tested it on.

**Comment by timb07** [comment]
The only valid PDFs I've found that trigger this bug contain private information and so I can't share them. However, I've found an invalid PDF from some spam that triggers it. (When the bug is fixed, a different error (caused by the PDF being badly formed) is returned instead: `pdfminer.psparser.PSEOF: Unexpected EOF`.) This file is attached.

[AWARD NOTIFICATION.pdf](https://github.com/pdfminer/pdfminer.six/files/1870028/AWARD.NOTIFICATION.pdf)
